dev-lang/julia-bin/julia-bin-1.0.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+MY_PN=${PN/-bin/}
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="High-performance programming language for technical computing"
+HOMEPAGE="https://julialang.org/"
+SRC_URI="
+	x86? ( https://julialang-s3.julialang.org/bin/linux/x86/1.0/${MY_P}-linux-i686.tar.gz )
```

Comment:
I'm assuming these to be shared linked against glibc, correct? Probably a good idea to mask
them on musl profiles or require `elibc_glibc`/`elibc_FreeBSD`.